cra-server/router/routes.js

```diff
@@ -18,13 +18,17 @@ const loginLimiter = rateLimit({
   max: 100, // limit each IP to 100 requests per windowMs
 });
 
-const cert = fs.readFileSync(
-  path.join(__dirname, "../../") + "ssl/keys/server.cert"
+// used for client authentication (so we can trust the server)
```

Comment:
I think these comments are inverted at least in terminoligy -- the keystore is where you'd look for a certificate to send to the other party... it's the truststore that we would go to, to find out who we trust - via a CA or cert. So from the filename (keystore) I think you mean 'so the server can trust us'. but will continue reading ..